inc/editor/namespace.php

```diff
@@ -515,6 +515,27 @@ function show_kitchen_sink( $args ) {
 	return $args;
 }
 
+function mce_settings( $settings, $editor_id ) {
+	$settings['init_instance_callback'] = <<<JS
```

Comment:
I tried to make this in a separate file, but babel will complain when compiling:
```
Module build failed (from ./node_modules/babel-loader/lib/index.js):
SyntaxError: /app/web/app/plugins/pressbooks/assets/src/scripts/tinymce-settings.js: Unexpected token (1:8)

> 1 | function(editor) {
    |         ^
```
and that because it needs a function name (or export it). I tried some things like exporting the method and overriding `tinyMCE.init` to use this new exported function instead of the WP way (`tiny_mce_before_init` hook).
I also tried to avoid webpack compilation and read the raw file. But at the end, given the size and low complexity of this JS lines, I decided to leave it as it is.